ui/admin/app/styles/app.scss

```diff
@@ -698,3 +698,23 @@
   min-height: 1rem;
   margin: 1.125rem 0 1.5rem;
 }
+
+.mapping-list {
+  width: 100%;
+
+  li {
+    padding: sizing.rems(xs) 0;
+    display: grid;
+    grid-template-columns: 1fr 1fr 5%;
+    align-items: center;
+    gap: sizing.rems(l);
+
+    > div {
```

Comment:
Can this strictly use class names rather than elements?  It's a bit more flexible.

Reply:
Updated to use class names, the div one I changed to `*` to make it clear it's to clear any existing bottom margin (mainly from HDS)